src/main/kotlin/com/devfloor/untitled/article/presentation/ArticleController.kt

```diff
@@ -0,0 +1,19 @@
+package com.devfloor.untitled.article.presentation
+
+import com.devfloor.untitled.article.application.ArticleFacade
+import com.devfloor.untitled.article.application.ArticleResponse
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class ArticleController(
+    private val articleFacade: ArticleFacade
+) {
+    @GetMapping("/articles/{id}")
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
```suggestion
    @ResponseStatus(value = HttpStatus.OK)
```

이런 식으로 쓰는 거 어때? 여기 매개변수로 들어가는 값이 여러개라 이렇게 써주면 뭔가 편할 것 같기도?!

Reply:
반영완료 !

Reply:
그냥 저렇게 매개변수 전달하는 부분은 앞으로 전부 값 명시해주는거로 통일하는게 어때?
어디는 쓰고 어디는 안쓰고 이렇게하면 헷갈릴거같으니

아 그리고 메서드 반환형도 (간단한 경우에도)항상 명시하는거로 통일하는게 좋을거같아
전부 명시 안하는건 절대 안되기 때문에, 전부 명시 하는거로 해서 통일성있게 가는게 좋을거같아.

Reply:
이 부분에 대한 논의 slack에 남겨놓았음 !